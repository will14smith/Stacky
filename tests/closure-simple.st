// closures capture by value, even though `a` is not on the stack it should still be available when invoked
main () -> () { 1 (a) { { a } } invoke print }