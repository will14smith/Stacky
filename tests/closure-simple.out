=== Output ===
1

=== Exit ===
0
